### PR TITLE
BETA: Register lens.is-a.dev

### PR DESCRIPTION
**Pull request by imlens**
Added `lens.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/lens.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "imlens",
+        "email": "xlunarr@hotmail.com"
+    },
+    "record": {
+        "CNAME": "lens"
```
Comment (DEV-DIBSTER) [inline domains/lens.json:7]: Invalid CNAME record.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/lens.json (hunk 1 above)
Comment (imlens) [reply domains/lens.json:7]: what was the CNAME supposed to be again?

Comment (DEV-DIBSTER) [reply domains/lens.json:7]: A valid hostname without any URL schemes attached.

E.g: beta.dibster.com

Comment (imlens) [reply domains/lens.json:7]: would imlens.github.io work?

**Pull request closed without merging**